### ISSUE TITLE
How do you enable webpack warning details?

### ISSUE THREAD

**Issue by nathangross** [tighten/jigsaw#538]
I'm getting a webpack warning:

```
1 WARNING in child compilations
webpack compiled with 1 warning
```

But I'm not sure where I can see the full warning. I do see an exerpt of the warning in my notifications panel (macOS) but looking for the full output. 

<img width="352" alt="Screen Shot 2021-01-28 at 12 29 57 PM" src="https://user-images.githubusercontent.com/380278/106175802-8ae7c100-6164-11eb-8e41-f3bde2b912df.png">

I've (somewhat randomly) tried: 
```
mix.webpackConfig({
    stats: {
        warnings: true,
    }
});
```

**Comment by bakerkretzmar** [comment]
Can you share your full `webpack.mix.js` and the command you're running?

**Comment by nathangross** [comment]
```
const mix = require('laravel-mix');
require('laravel-mix-jigsaw');

// I just added this hoping that it might show full warnings (found on a somewhat related post)
mix.webpackConfig({
    stats: {
        warnings: true,
    }
});

mix.disableSuccessNotifications();
mix.setPublicPath('source/assets/build');

mix.jigsaw()
    .js('source/_assets/js/main.js', 'js')
    .postCss('source/_assets/css/main.css', 'css', [
        require('tailwindcss'),
    ])
    .version();
    ```
    
Pretty sure it has to do with postcss and something in a 3rd party css file I'm using. Maybe an image url in the file. But just having a hard time tracking it down without the full warning message.

**Comment by bakerkretzmar** [comment]
What's the full command you're running (if it's an npm script, what's the content of the script)? What version of Mix are you on? Does your build succeed and output this warning or is it actually failing to compile?

Is the site you're working on public by any chance? I want to try this locally, I have a feeling it's gonna be hard to diagnose unless I can run it and poke around myself. If it's not public but you're able to share a repo with a minimal reproduction that would also be super helpful.

**Comment by nathangross** [comment]
I'm running `npm run watch`

```
    "scripts": {
        "dev": "mix",
        "watch": "mix watch",
        "staging": "NODE_ENV=staging mix",
        "production": "mix --production"
    },
```

The build succeeds but says:

```
✔ Compiled Successfully in 4358ms
/js/main.js │ 6.47 KiB │
css/main.css │ 3.98 MiB │

1 WARNING in child compilations
webpack compiled with 1 warning
```

webpack 5 and mix 6

```
    "devDependencies": {
        "cross-env": "^3.2.3",
        "laravel-mix": "^6.0.10",
        "laravel-mix-jigsaw": "^1.5.0",
        "postcss-import": "^14.0.0",
        "postcss-loader": "^4.1.0",
        "tailwindcss": "^2.0.2",
        "tailwindcss-textshadow": "^2.1.3",
        "webpack": "^5.16.0"
    },
```

It's definitely something in a css file I'm importing:
```
@import "./vendor/royalslider.css";

// If I comment this file out, it compiles fine with no warnings.
@import "./vendor/rs-default.css";
```

**Comment by bakerkretzmar** [comment]
Interesting, thanks.

Try adding this to your mix file:

```diff
mix.jigsaw()
    .js('source/_assets/js/main.js', 'js')
    .postCss('source/_assets/css/main.css', 'css', [
        require('tailwindcss'),
    ])
+   .options({ processCssUrls: false })
    .version();
```

That might mess with other URLs, if it does, you could try disabling URL rewrites for just that file, which I didn't even know was a thing: https://laravel-mix.com/docs/6.0/url-rewriting#per-file-rewrite-settings

You may need to separate it out of `main.css` and have another file that you build via Mix that just imports that one vendor file, so that you can target just that one.

**Comment by nathangross** [comment]
Hmm, I added that option but still same result/warning. I was hoping that'd work since I was pretty sure it has to do with urls in that css file.

**Comment by nathangross** [comment]
Is there a log file created anywhere that might provide the whole warning message?

The repo is private—but happy to add you to it if you want.

**Comment by bakerkretzmar** [comment]
Hmm, okay. I don't think there is a log file, no, but you could try `mix -v` or `mix -- --verbose` to get more output in the console. Can you share just `vendor/rs-default.css` and I'll try a couple more things locally? Otherwise yeah if you don't mind, I'm interested to see what's going on.

**Comment by nathangross** [comment]
I've added you to the repo. I appreciate your help!

**Comment by bakerkretzmar** [comment]
Thanks! It's the linear gradients in `rs-default.css`—autoprefixer says they're using old/incorrect direction syntax, when I replaced `gradient(top` with `gradient(to bottom` in that file (4 lines) no warnings were generated.

To see the full warning I ran `npx mix -- --stats=verbose`, I didn't know about this but it looks like `stats` accepts way more than just true/false: https://webpack.js.org/configuration/stats/#stats-presets. I assume setting any of these in your Mix file would work too.
